src/config/configAggregator.ts

```diff
@@ -321,11 +324,7 @@ export class ConfigAggregator extends AsyncOptionalCreatable<JsonMap> {
    * Add an allowed config property.
    */
   public addAllowedProperties(configMetas: ConfigPropertyMeta | ConfigPropertyMeta[]): void {
-    if (isArray(configMetas)) {
-      this.allowedProperties.push(...configMetas);
-    } else {
-      this.allowedProperties.push(configMetas);
-    }
+    this.allowedProperties.push(...(isArray(configMetas) ? configMetas : [configMetas]));
```

Comment:
Why this change? It doesn't change the functionality but makes the code harder to follow.

Is there a performance benefit or something like that?

Reply:
reverted